upgrade/1.5.2/CSM-Only.md

```diff
@@ -0,0 +1,146 @@
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-kubernetes")) | $o[.].id')"
+      export KUBERNETES_IMAGE_ID
+      echo "KUBERNETES_IMAGE_ID=$KUBERNETES_IMAGE_ID"
+      ```
+
+    * Storage-CEPH ID
+
+      ```bash
+      STORAGE_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-storage")) | $o[.].id')"
+      export STORAGE_IMAGE_ID
+      echo "STORAGE_IMAGE_ID=$STORAGE_IMAGE_ID"
+      ```
+
+1. (`ncn-m001#`) Create CFS sessions for both Kubernetes and Storage nodes.
+
+    1. Set variables with unique names for the CFS sessions to be created.
+
+       ```bash
+       TSTAMP=$(date "+%Y%m%d%H%M%S")
+       K8S_CFS_SESSION_NAME="management-k8s-${CSM_RELEASE}-${TSTAMP}"
+       CEPH_CFS_SESSION_NAME="management-ceph-${CSM_RELEASE}-${TSTAMP}"
+       ```
+
+    1. Build customized master node images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Master \
+           "$KUBERNETES_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map "$KUBERNETES_IMAGE_ID" "management-kubernetes-${CSM_RELEASE}" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "${K8S_CFS_SESSION_NAME}" \
+           --format json
+       ```
+
+    1. Build customized storage node images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Storage \
+           "$STORAGE_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map "$STORAGE_IMAGE_ID" "management-storage-${CSM_RELEASE}" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "${CEPH_CFS_SESSION_NAME}" \
+           --format json
+       ```
+
+1. (`ncn-m001#`) Wait for the image builds to complete successfully.
+
+   > This may take over 20 minutes to complete.
+
+   ```bash
+   watch "
+   cray cfs sessions describe "${K8S_CFS_SESSION_NAME}" --format json | jq -r ".status.session.succeeded";
+   cray cfs sessions describe "${CEPH_CFS_SESSION_NAME}" --format json | jq -r ".status.session.succeeded"
+   "
+   ```
+
+   Expected results:
+
+   ```text
+   true
+   true
+   ```
+
+1. (`ncn-m001#`) Set environment variables with the IMS image IDs.
+
+    * Kubernetes image IMS ID
+
+      ```bash
+      NEW_KUBERNETES_IMAGE_ID="$(cray cfs sessions describe "${K8S_CFS_SESSION_NAME}" --format json | jq -r '.status.artifacts[].result_id')"
+      export NEW_KUBERNETES_IMAGE_ID
+      echo "NEW_KUBERNETES_IMAGE_ID=$NEW_KUBERNETES_IMAGE_ID"
+      ```
+
+    * Storage image IMS ID
+
+      ```bash
+      NEW_STORAGE_IMAGE_ID="$(cray cfs sessions describe "${CEPH_CFS_SESSION_NAME}" --format json | jq -r '.status.artifacts[].result_id')"
+      export NEW_STORAGE_IMAGE_ID
```

Comment:
Do we need to export this? Looks like we're passing it into the script as an argument.

```suggestion
```